packages/core/src/config/config.ts

```diff
@@ -89,9 +89,12 @@ export interface StoreCurrency {
     thousandsSeparator: string;
 }
 
+export type UserExperienceSettingNames = 'walletButtonsOnTop';
+
 export interface CheckoutSettings {
     features: { [featureName: string]: boolean };
     checkoutBillingSameAsShippingEnabled: boolean;
+    CheckoutUserExperienceSettings: { [key in UserExperienceSettingNames]: boolean };
```

Comment:
I have updated the code based on ther server side PR. 🙇🏻‍♂️
cc @PascalZajac @animesh1987